.gitignore

```diff
@@ -98,3 +98,6 @@ scripts/hide_extra_registration_logs.json
 # Celery dbs
 celerybeat-schedule.db
 celery_beat.py.db
+
+# Diff (used during merge resolution)
+*.orig
```

Comment:
This is fine, but things like this ought to go in a global gitignore: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore